### DISCUSSION TITLE
SolveLinearSystem's operation during a simulation

### DISCUSSION THREAD

**Discussion by we85** [SCIInstitute/SCIRun#2331] [Q&A]
Hi all,

I would like to know if it is possible to observe how the SolveLinearSystem module converges to a solution during a simulation? (As in, how the error decreases with each iteration)

Thank you in advance!

**Comment by jessdtate** [comment]
This is a feature that was in SCIRun 4, yet hasn't been ported to SCIRun 5.   However, there are enough pieces in place that we should be able to show a graph at the end of the execution (not updated continuously like in SCIRun 4) at least, or a similar solution.  We will assess when it could be implemented
